azure-pipelines.yml

```diff
@@ -0,0 +1,56 @@
+pool:
+  # self-hosted agent on Windows 10 1809+ environment
+  # includes newer Docker engine with LCOW enabled, new build of LCOW image
+  # includes Ruby 2.5, Go 1.10, Node.js 10.10, hadolint
+  name: Default
+
+variables:
+  NAMESPACE: puppet
+
+steps:
+- checkout: self  # self represents the repo where the initial Pipelines YAML file was found
+  clean: true  # whether to fetch clean each time
+- powershell: |
+    $gemfile = Join-Path -Path (Get-Location) -ChildPath 'docker/Gemfile'
+    $gempath = Join-Path -Path (Get-Location) -ChildPath 'docker/.bundle/gems'
+    bundle config --local gemfile $gemfile
+    bundle config --local path $gempath
+    bundle install
+  displayName: Fetch Dependencies
+  name: fetch_deps
+- powershell: |
+    . "$(bundle show pupperware)/ci/build.ps1"
+    Write-HostDiagnostics
+  displayName: Diagnostic Host Information
+  name: hostinfo
+- powershell: |
+    . "$(bundle show pupperware)/ci/build.ps1"
+    Lint-Dockerfile -Name puppet-runtime
+  displayName: Lint Dockerfile
+  name: lint_dockerfile
+- powershell: |
+    . "$(bundle show pupperware)/ci/build.ps1"
+    Build-Container -Name puppet-runtime -Namespace $ENV:NAMESPACE
+  displayName: Build Container
+  name: build_container
+ - powershell: |
```

Comment:
Oops, extra space at beginning of line here.

Nice to see how easy this Azure support was to add!!